raiden/ui/app.py

```diff
@@ -418,10 +420,11 @@ def run_app(
 
     service_registry = None
     if routing_mode == RoutingMode.PFS:
+        services_contracts = services_deployment_data['contracts']
```

Comment:
This means that using PFS in **production** mode will result in "used-before-declaration", no?

Reply:
Also, `service_contracts` is set twice. First one would be in line 333

Reply:
Hmm I have removed this in the last commit.

Reply:
As I said this is removed in the last commit, github must have shown you an older state?